### ISSUE TITLE
Headset bug related with FM BT Radio

### ISSUE THREAD

**Issue by LeopardSurd** [CyanogenDefy/android_device_motorola_jordan#51]
Here is how you can duplicate it:

Open the BT FM Radio with the headset plugin. Stop the BT FM Radio app normaly and start the music player (any player): the music will play through external speaker. In the same condition, instead of opening a player, I open the ported FM Radio (from Droid)...the app start searching the headset, and searching is indefinitely in time. Meanwhile, the status bar shows that the headset are connected.

Plug out/in of the headset doesn't solve the problem. Only the reboot.

Tested on last nightly: 8 July.

**Comment by maniac103** [comment]
Which of the two radio apps did you reproduce this with? The one with the orange icon is known to not be ready yet...

**Comment by LeopardSurd** [comment]
Yes, is the one with the orange button (that supports Bluetooth) not the ported one (black/red). I know that is not finished yet but it's a strange bug. :)
